llama_hub/llama_packs/panel_chatbot/app.py

```diff
@@ -0,0 +1,371 @@
+"""Provides a ChatBot UI for a Github Repository. Powered by Llama Index and Panel"""
```

Comment:
I've been considering whether to contribute a *Hello World" app.py file, a *Medium Example* app.py file or a *Very Advanced Example* app.py file. I settled on a *Medium Example* that I actually find useful as its own tool.

Do you have any thoughts on what to include?

Reply:
the medium app.py example is great

Reply:
FWIW, I think Medium is the right level :)